src/main.rs

```diff
@@ -187,6 +201,39 @@ fn main() -> anyhow::Result<()> {
     }
 }
 
+// Argument to the generate_versioned_crates function.
+enum CurrentCratePath<'a> {
+    CurrentRustdocPath(&'a Path), // If rustdoc is passed, it is just loaded into the memory.
+    ManifestPath(&'a Path),       // Otherwise, the function generates the rustdoc.
```

Comment:
`the function` -- which function? Reading the file from top to bottom, it's not clear that it mentions the `generate_versioned_crates` function.

Reply:
How so? It is mentioned in the comment above

Reply:
Oh, I missed this comment. Thanks.